include/boost/json/conversion.hpp

```diff
@@ -0,0 +1,293 @@
+//
+// Copyright (c) 2022 Dmitry Arkhipov (grisumbras@yandex.ru)
+//
+// Distributed under the Boost Software License, Version 1.0. (See accompanying
+// file LICENSE_1_0.txt or copy at http://www.boost.org/LICENSE_1_0.txt)
+//
+// Official repository: https://github.com/boostorg/json
+//
+
+#ifndef BOOST_JSON_CONVERSION_HPP
+#define BOOST_JSON_CONVERSION_HPP
+
+#include <boost/json/string_view.hpp>
+#include <boost/mp11/algorithm.hpp>
+#include <boost/mp11/utility.hpp>
+
+#include <utility>
+
+BOOST_JSON_NS_BEGIN
+namespace detail {
```

Comment:
Implementation details should be in json/detail or json/impl not a public header since this is scanned by doxygen

Reply:
This stuff should go in `<boost/json/detail/conversion.hpp>`